Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/AbilData.xml

```diff
@@ -6607,6 +6607,21 @@
         <AINotifyEffect value=""/>
         <CmdButtonArray index="Execute" DefaultButtonFace="GuardianShield"/>
     </CAbilEffectInstant>
+    <CAbilEffectInstant id="AP_ActivateSquadSight">
+        <EditorCategories value="Race:Protoss,AbilityorEffectType:Units"/>
+        <Effect index="0" value="AP_SquadSightPersistent"/>
+        <Flags index="AllowMovement" value="1"/>
+        <!-- <Flags index="AutoCast" value="1"/> -->

```

Comment:
That requires an autocast validator, which I didn't want to make. I suppose I can just copy the guardian shell one, and it's fine if it isn't great because of the off-by-default.